frontend-react/src/pages/misc/FeatureFlags.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { _exportForTesting, CheckFeatureFlag } from "./FeatureFlags";
```

Comment:
Is `CheckFeatureFlag `only intended for use inside the `FeatureFlags` component and the test file? 
It could be grouped with `_exportForTesting`.

Reply:
Good question: `CheckFeatureFlag()` is the only function that should be used in the rest of the project, that's why it's at the root.

Exporting the other functions inside `_exportForTesting` is so a `FeatureFlags.test.tsx` can still reach them to implement unit tests. It's not the best solution, but it works and isn't too clever.